frontend/lib/tests/fake-base-field-props.ts

```diff
@@ -0,0 +1,11 @@
+import { BaseFormFieldProps } from "../form-fields";
+
+export function fakeBaseFieldProps<T>(props: Partial<BaseFormFieldProps<T>> & { value: T }): BaseFormFieldProps<T> {
+  return {
+    onChange: jest.fn(),
+    name: 'foo',
+    id: 'foo',
+    isDisabled: false,
+    ...props
+  };
+}
```

Comment:
I moved this to its own file so changes to tests in `form-fields.test.tsx` wouldn't confusingly trigger other test suites to run.